autoagora/utils/constants.py

```diff
@@ -14,6 +14,13 @@
 # stddev time:  {{frequent_query.stddev_time}}
 {{frequent_query.query}} => {{frequent_query.avg_time}} * $GLOBAL_COST_MULTIPLIER;
 {% endfor %}
+{% if manual_entry is defined %}
 
-default => $DEFAULT_COST * $GLOBAL_COST_MULTIPLIER;\
+{{manual_entry}}
+
+default => $DEFAULT_COST * $GLOBAL_COST_MULTIPLIER;
+{% else %}
+default => $DEFAULT_COST * $GLOBAL_COST_MULTIPLIER;
+{% endif %}\
 """
+MANUAL_AGORA_MODEL_PATH = "/etc/manual_agora_entry.txt"
```

Comment:
That not how you would normally use `/etc`. According to the POSIX spec and common practice, you'd store those files under `/etc/opt/autoagora/agora_manual_entries/[subgraph_hash].agora`. Note that we have to have one file per subgraph.
In any case, we can probably simplify things, and stick to the XDG standard. I found https://github.com/platformdirs/platformdirs which will make it easy. Perhaps we should use `user_config_dir` instead of `site_config_dir`, as it should be a bit easier to deal with by the user (no root required).

Reply:
Nice, this looks really nice will add this!